server/controllers/scheduledJobsController.js

```diff
@@ -0,0 +1,37 @@
+import schedule from "node-schedule";
+
+import firestore from "../db.js";
+import emailService from "../services/emailService.js";
+
+import transactionService from "../services/transactionService.js";
+import userService from "../services/userService.js";
+
+const scheduledJobsController = {
+
+    // Schedules a job to send the reminder to return the borrowed dish
+    dishCheckoutReturnReminder: async function (transId) {
```

Comment:
Sorry, would we be able to do this as a batch job?

I.e. every Friday at 2:00pm check for transactions that are minimum N-days old and send an email to the users for them all at once (that way we can batch multiple missing dishes into one email together)

Reply:
Updated it to a batch job. The config needs to be in cron syntax for it to be valid.